src/utils/aiprdescription/cursorPositionInsert.ts

```diff
@@ -15,5 +27,5 @@ export const insertAtCursorFromStream = async (textArea: HTMLTextAreaElement, st
       textArea.setRangeText(chunk, start, end, "end");
     }
   }
-  textArea.setRangeText("\n\n_Description generated using [OpenSauced](https://opensauced.ai/_).", textArea.selectionStart, textArea.selectionEnd, "end");
-};
+  textArea.setRangeText("\n\n_Description generated using [OpenSauced](https://opensauced.ai/)._", textArea.selectionStart, textArea.selectionEnd, "end");
```

Comment:
same here. (to make this reusable)

Reply:
Done. dea5ce1d6562ab4604f2e49aabf28caa4f024c49.